utils/yoke/src/yoke.rs

```diff
@@ -650,12 +643,107 @@ impl<Y: for<'a> Yokeable<'a>, C> Yoke<Y, C> {
         }
     }
 
-    /// This is similar to [`Yoke::map_project`], but it works around it not being able to
-    /// use `FnOnce` by using an explicit capture input.
+    /// This is similar to [`Yoke::map_project`], however it can also bubble up an error
+    /// from the callback.
+    ///
+    /// This is a bit more efficient than cloning the [`Yoke`] and then calling [`Yoke::map_project`]
```

Comment:
Nit: The docs don't seem to match the function

Reply:
What I meant was that the "bit more efficient than..." paragraph belongs only in the `_cloned` functions, which this one is not.